docs/releases/core/release-notes.md

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_position: 1
+title: Core software releases
+hide_title: false
+---
+
+While the Vega core software is closed-source, you can refer here for a full list of release notes for each version that the validators use to run the Vega mainnet. Releases are listed with their semantic version number and the date the release was made available to mainnet validators.
+
+### Versions 0.46.0-0.47.4 combined | 2022-01-11
+
+This release was made available to validators on 11 January, 2022. 
+
+A key theme of this combined release has been improvements to the checkpointing feature; this includes fixes to ensure epochs and other key data is preserved as they should be during checkpoint restarts. In addition to this, the “free-form governance” feature has been implemented. This feature further decentralises the protocol by allowing users to submit a range of governance proposals for community consideration and voting.
+
+TThe protocol calculates a validator score for each validator. This score is used to set their voting power in Tendermint and determine their reward amounts. The changes introduced in this release mean that the protocol no longer prevents users from delegating to any node, however, an overcrowded node will impact the validator score thus affecting rewards. Further scoring of performance measurements in future releases will bring with it the mechanism to adjust rewards in reflection of validator performance in the network. This is another step in getting the network more decentralised, and open to new validators joining the network.
```

Comment:
@candida-d just noticed a double T at the start of the paragraph

Reply:
Oops! Thanks @gordsport